src/userConfig_sample.h

```diff
@@ -58,11 +58,14 @@ enum MACHINE {
 #define BREWDETECTION_TYPE 1                     // 1 = Software (Onlypid 1), 2 = Hardware (Onlypid 0), 3 = optocoupler for Only PID
 #define FEATURE_POWERSWITCH 0                    // 0 = deactivated, 1 = activated
 #define POWERSWITCH_TYPE Switch::TOGGLE          // Switch::TOGGLE or Switch::MOMENTARY (trigger)
+#define POWERSWITCH_MODE Switch::NORMALLY_OPEN   // Switch::NORMALLY_OPEN or Switch::NORMALLY_CLOSED
 #define FEATURE_BREWSWITCH 0                     // 0 = deactivated, 1 = activated
 #define BREWSWITCH_TYPE Switch::TOGGLE           // Switch::TOGGLE or Switch::MOMENTARY (trigger)
+#define BREWSWITCH_MODE Switch::NORMALLY_OPEN    // Switch::NORMALLY_OPEN or Switch::NORMALLY_CLOSED
 #define FEATURE_STEAMSWITCH 0                    // 0 = deactivated, 1 = activated
 #define STEAMSWITCH_TYPE Switch::TOGGLE          // Switch::TOGGLE or Switch::MOMENTARY (trigger)
 #define OPTOCOUPLER_TYPE HIGH                    // BREWDETECTION 3 configuration; HIGH or LOW trigger optocoupler
+#define STEAMSWITCH_MODE Switch::NORMALLY_OPEN   // Switch::NORMALLY_OPEN or Switch::NORMALLY_CLOSED
```

Comment:
should be behind steamswitch_type

Reply:
I had already removed the OPTOCOUPLER stuff and added it back in before creating the PR ;)

Reply:
:D :D